webapp/src/components/backstage/playbook_runs/playbook_run_list/playbook_run_list.tsx

```diff
@@ -428,7 +428,7 @@ const BackstagePlaybookRunList = () => {
                                 id={playbookRun.id}
                                 text={playbookRun.name}
                             />
-                            <TeamName>{teams.length > 1 ? getTeamName(teams, playbookRun.team_id) : ''}</TeamName>
+                            {teams.length > 1 ? <TeamName>{getTeamName(teams, playbookRun.team_id)}</TeamName> : ''}
```

Comment:
Same as above -- suggest:
```suggestion
                            {teams.length > 1 && <TeamName>{getTeamName(teams, playbookRun.team_id)}</TeamName>}
```